### PR TITLE
Consolidate VMI Performance categories

### PR DESCRIPTION
**Pull request by andrewki-msft**
Consolidate VMI Performance workbooks into a single category to de-clutter the workbooks gallery

### CODE REVIEW

**Comment by rakshith210** [comment]
Could you add a screenshot of it would look in the gallery ?

File: Workbooks/Virtual Machine Scale Sets - Performance Analysis/Performance Analysis (Perf)/settings.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
     "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
-    "name":"Performance",
+    "name":"Perf Counters",
     "author": "Microsoft",
     "galleries": [
-        { "type": "workbook", "resourceType": "microsoft.compute/virtualmachinescalesets", "categoryKey": "vmInsightsVirtualMachinesPerf", "order": 200 },
-        { "type": "insights", "resourceType": "microsoft.compute/virtualmachinescalesets", "categoryKey": "vmInsightsVirtualMachinesPerf", "order": 200 }
+        { "type": "workbook", "resourceType": "microsoft.compute/virtualmachinescalesets", "categoryKey": "vmInsightsVirtualMachines", "order": 210 },
```
Comment (yves-chan) [inline Workbooks/Virtual Machine Scale Sets - Performance Analysis/Performance Analysis (Perf)/settings.json:6]: do we need to define the category key "vmInsightsVirtualMachines"?

Comment (andrewki-msft) [reply Workbooks/Virtual Machine Scale Sets - Performance Analysis/Performance Analysis (Perf)/settings.json:6]: Thanks @yves-chan, resolved.

**Comment by andrewki-msft** [comment]
Tried testing using **feature.workbookGalleryBranch=users/andrewki-msft/vmi-workbooks** but not seeing my changes being reflected. I don't think my type of changes would be reflected using this method so going ahead and merging this in.

**Pull request merged**